python/src/lib/workbook_validator.py

```diff
@@ -94,20 +129,22 @@ def validate_cover_sheet(
     errors = []
     project_schema = None
     cover_header = get_headers(cover_sheet, "A1:B1")
-    cover_row = map(lambda cell: cell.value, cover_sheet[2])
+    row_num = 2
+    cover_row = map(lambda cell: cell.value, cover_sheet[row_num])
     row_dict = map_values_to_headers(cover_header, cover_row)
     try:
         CoverSheetRow(**row_dict)
     except ValidationError as e:
-        errors.append(f"Cover Sheet: Invalid {e}")
+        erroring_column = get_erroring_column(CoverSheetRow, e)
+        errors.append(WorkbookError(f"{COVER_SHEET} Sheet: Invalid {e}", f"{row_num}", f"{erroring_column}", COVER_SHEET))
         return (errors, None)
 
         # This does not need to be a silent failure. This would be a critical error.
     project_schema = SCHEMA_BY_PROJECT[row_dict["Project Use Code"]]
     return (errors, project_schema)
 
 
-def validate_project_sheet(project_sheet: Worksheet, project_schema) -> Errors:
+def validate_project_sheet(project_sheet: Worksheet, project_schema, project_use_code) -> Errors:
```

Comment:
I _think_ `project_use_code` may not be necessary here. I believe `SCHEMA_BY_PROJECT[project_use_code] == project_schema`. If that is not the case, feel free to ignore the comment.

Reply:
Oooh I think you're right; I missed that since it was bubbled up from `validate_ cover_sheet`